### DISCUSSION TITLE
Drumkit file for marching percussion?

### DISCUSSION THREAD

**Discussion by duck57** [hydrogen-music/hydrogen#1130] [Q&A]
Are there any drumkits that represent marching percussion?  If not, how feasible would it be to create one based on [MuseScore's MDL module](https://musescore.org/en/mdl)?

**Comment by trebmuh** [comment]
Only pointing out to the [Contributing-drumkits wiki page](https://github.com/hydrogen-music/hydrogen/wiki/Contributing-drumkits) which is probably not up-to-date, but still a good first step I imagine.

**Comment by duck57** [comment]
I found the MDL repository and [made a post over on the MuseScore forum](https://musescore.org/en/node/316690) about the possibility of using the MDL sounds as the base for a Hydrogen drumkit.  I hope to start working on this when/if I have free time in the next month or two.
